libraries/ui/src/default-config/tailwind.css

```
@@ -319,71 +324,65 @@
     font-style: normal;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-style: normal
+    font-display: swap;
 }
 
 @font-face {
-    font-family: "Reckless Neue";
-    src: url("https://bluedot.org/fonts/RecklessNeue-LightItalic.woff2") format("woff2");
-    font-weight: 300;
+    font-family: "Inter";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)